README.md

```diff
@@ -43,6 +43,7 @@ please sort it in ascending order.
 
 ### Blog
 
+- [andriawan.com](https://github.com/andriawan/andriawan.com) oleh [Muhammad Irwan Andriawan](https://github.com/andriawan) - React, Gatsby
```

Comment:
`oleh` -> `by`

Reply:
sorry, my bad @jefrydco...i'll fix it soon